js/src/View/AppView.js

```diff
@@ -0,0 +1,87 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var UserModel = require('../Model/UserModel.js');
+var appViewTemplate = require('../templates/container.hbs');
+var TodoListView = require('./TodoListView.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoListModel = require('../Model/TodoListModel.js');
+var todoAppTemplate = require('../templates/todoapp.hbs');
+
+var AppView = Backbone.View.extend({
+    el: 'body',
+    events: {
+        'click .addlist': 'addNewList',
```

Comment:
Do you want to use the `js-` prefix convention we talked about to point out js hooks instead of doing camel casing? I'll just add comments asking you to update these kind of selectors to that format assuming yes. `js-add-list`